tests/integration/components/select-component-test.js

```diff
@@ -139,23 +152,61 @@ test('It displays the specified component when componentNameForGroupTooltip is p
   );
 });
 
-test('It does not display loading text and empty content component at the same time', async function(assert) {
-  this.container.register('template:components/some-component', hbs`<span data-test-some-component>No results</span>`);
+test(
+  'It does not display loading text and does show empty content component when content is empty and loaded',
+  async function(assert) {
+    this.container.register('view:some-view', Ember.View.extend({ templateName: 'some-view' }));
+    this.container.register('template:some-view', hbs`<span data-test-some-component>No results</span>`);
+
+    this.set('content', []);
+
+    await this.render(hbs`
+    {{select-component
+        content=content
+        emptyContentView='some-view'
+    }}`);
+
+    await click(this.helpers.selectChoiceElement);
+
+    assert.ok(!this.helpers.contentLoadingElement, 'Loading ui is not present');
+    assert.ok(!!this.helpers.contentEmptyElement, 'Empty content is present');
```

Comment:
why not just use something like `assert.dom('.portal-container').exists('message')` or `assert.dom('.portal-container').doesNotExist('message')`?
ditto for all the other occurrences here where we need to turn jquery into a boolean

Reply:
or if the page object above is a legit pageobject you can just use pageObject stuff, documented here http://ember-cli-page-object.js.org/docs/v1.15.x/

Reply:
I paired with Matt B on this so I'll let him answer this one

Reply:
I don't think this repo has https://github.com/simplabs/qunit-dom installed, and so `assert.dom` is not available.

This page object is just a class following the page object pattern. We didn't introduce a dependency on ember-cli-page-object.